custom/ewsghana/tests/handlers/utils.py

```diff
@@ -113,9 +121,16 @@ def setUpClass(cls):
                 caption='receipts'
             )
         )
-        config.consumption_config = ConsumptionConfig(min_transactions=0, min_window=0, optimal_window=60)
+        config.consumption_config = ConsumptionConfig(use_supply_point_type_default_consumption=True)
```

Comment:
To simplify tests we use manual consumption (manual consumption is also used in v1 tests)